desc/compute/_field.py

```diff
@@ -2160,6 +2220,28 @@ def _B_sub_zeta_rz(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="B0",
+    label="B_0",
+    units="T",
+    units_long="Tesla",
+    description="Average magnitude of magnetic field on the magnetic axis",
+    dim=0,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="",
+    data=["|B|", "sqrt(g)"],
+)
+def _B0(params, transforms, profiles, data, **kwargs):
+    # indices of the innermost flux surface
+    idx = jnp.where(transforms["grid"].inverse_rho_idx == 0)[0]
```

Comment:
check if this works with JAX and see surface integral code if it does not

Reply:
`sum((transforms["grid"].inverse_rho_idx == 0) * integrand)`